src/customer/guest-credentials.ts

```diff
@@ -1,5 +1,6 @@
-export default interface GuestCredentials {
+import { Subscriptions } from '../subscription';
+
+export type GuestCredentials = Partial<Subscriptions> & {
     id?: string;
     email: string;
-    marketingEmailConsent?: boolean;
```

Comment:
since this was an experimental feature, I have decided to remove it to reduce the code that we have to maintain.